modules/core/osgi-service-tracker-collections-test/src/testIntegration/java/com/liferay/osgi/service/tracker/collections/list/test/ServiceTrackerListTest.java

```diff
@@ -69,6 +70,33 @@ public void tearDown() {
 		}
 	}
 
+	@Test
+	public void testGetListAndArray() {
+		try (ServiceTrackerList<TrackedOne> serviceTrackerList =
+				ServiceTrackerListFactory.open(
+					_bundleContext, TrackedOne.class)) {
+
+			TrackedOne[] trackedOnes = {new TrackedOne(), new TrackedOne()};
+
+			Collection<ServiceRegistration<TrackedOne>> serviceRegistrations =
+				registerServices(TrackedOne.class, trackedOnes);
+
+			List<TrackedOne> trackedOneList = serviceTrackerList.toList();
+
+			Assert.assertArrayEquals(
+				serviceTrackerList.toString(), trackedOnes,
+				trackedOneList.toArray(new TrackedOne[0]));
+
+			TrackedOne[] trackedOnes1 = serviceTrackerList.toArray(
```

Comment:
@W2jx When there's 1, there's 2. Please rename this, maybe call it `trackedOneArray`.

Reply:
I will fix them.